app/src/main/java/com/example/drawingactivity/DialogUtils.kt

```diff
@@ -137,6 +135,196 @@ object DialogUtils {
             }
         }
     }
+
+    fun colorPickerBottomSheet(
```

Comment:
Please clean and separate this method a bit please. It's hard to follow and has functions in function

Reply:
cleaned it up a little more, might still need more work but I think it is in a much better spot now